.github/workflows/virtualblox.yml

```diff
@@ -0,0 +1,27 @@
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        sudo apt update && sudo apt install -y libsdl2-dev
+        git clone https://github.com/emscripten-core/emsdk.git
+        cd emsdk
+        ./emsdk install latest
+        ./emsdk activate latest
+
+    - name: Build VirtualBlox
+      working-directory: ./VirtualBlox
+      run: |
+        source ../emsdk/emsdk_env.sh
+        make -j all
```

Comment:
this is great thanks for adding this, similar to our main.yml action which we can probably rename now to tests.yml or blockware-build.yml

Reply:
spitballing now something to address in this phab, i’m happy to do this but it could also be nice to break up so each blockware generates its own action so they can fail specifically ?

Reply:
Yeah right now you have to look at the logs to see what failed. But there's a little bit of a maintenance tension. You can either:
* Make separate steps here and make sure you keep the list of steps here in sync with the Makefile
* Don't make separate steps here, and there's one fewer place to keep the list of targets in sync. I don't think there's a clean way to automatically split out every target in the Makefile into separate GH action steps

Reply:
But what I want to do is add the virtualblox targets so they run from platformio in each blockware target. So maybe we could just break out the list of each blockware here once and run all the targets for each blockware.